apps/src/music/player/MusicLibrary.ts

```diff
@@ -244,9 +244,9 @@ export default class MusicLibrary {
     }
     const folder = this.getFolderForFolderId(this.currentPackId);
     // Read key from the folder, or the first sound that has a key if not present on the folder.
-    return (
-      folder?.key || folder?.sounds.find(sound => sound.key !== undefined)?.key
-    );
+    return folder?.key !== undefined
+      ? folder?.key
+      : folder?.sounds.find(sound => sound.key !== undefined)?.key;
```

Comment:
nullish coalescing operator for the win!

```
folder?.key ?? folder?.sounds.find(sound => sound.key !== undefined)?.key
```

Lets us keep the simpler logic w/o adding the ternary.

Reply:
Nice, thanks!  Done.